source/code-generation/query-converter/bulk-conversion.txt

```diff
@@ -0,0 +1,93 @@
+.. _rm-bulk-conversion:
+
+===============
+Bulk Conversion
+===============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can bulk convert one or more queries, stored procedures, triggers, 
+and views in bulk by enabling the multi-select toggle. Copy and paste 
+your queries into the query converter individually and then bulk convert
+a selection to update them to work with MongoDB and your migrated schema.
+
+About this Task
+---------------
+
+.. include:: /includes/fact-query-converter-generic.rst
```

Comment:
(suggestion)
For the include. I don't think we need to define AI. I know this isn't your content, but consider removing the hover help here.